### PR TITLE
docs(logging,monitoring): Update platform-logging and monitoring docs

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: src/managing-workflow/platform-logging.md [hunk 1]
```diff
@@ -2,44 +2,16 @@
 
 The logging platform is made up of 2 components - [Fluentd](https://github.com/deis/fluentd) and [Logger](https://github.com/deis/logger).
 
-[Fluentd](https://github.com/deis/fluentd) runs on every worker node of the cluster and is deployed as a [Daemon Set](http://kubernetes.io/v1.1/docs/admin/daemons.html). The Fluentd pods capture all of the stderr and stdout streams of every container running on the host (even those not hosted directly by kubernetes). It then sends this data via the Syslog UDP port (514) to the Logger component.
+[Fluentd](https://github.com/deis/fluentd) runs on every worker node of the cluster and is deployed as a [Daemons Set](http://kubernetes.io/v1.1/docs/admin/daemons.html). The Fluentd pods capture all of the stderr and stdout streams of every container running on the host (even those not hosted directly by kubernetes). Once the log message arrives in our [custom fluentd plugin](https://github.com/deis/fluentd/tree/master/rootfs/opt/fluentd/deis-output) we determine where the message originated.
```
Comment (arschles) [inline src/managing-workflow/platform-logging.md]: On the first line, `s/Daemons Set/DaemonSet`

File: src/managing-workflow/platform-logging.md [hunk 2]
```diff
@@ -2,44 +2,16 @@
...
 
-Logger acts like a syslog server and receives all log messages that are occurring on the cluster. It then filters this data to only Deis deploy applications and stores those log messages in a ring buffer where they can be fetched via the Deis CLI.
+If the message was from the [Workflow Controller](https://github.com/deis/controller) or from an application deployed via workflow we send it to a local [NSQD](nsq.io) instance putting it in the logs topic.
```
Comment (arschles) [inline src/managing-workflow/platform-logging.md]: probably clearer to say "...we send it to the `logs` topic on the local [NSQD](nsq.io) instance."

File: src/managing-workflow/platform-logging.md [hunk 3]
```diff
@@ -50,38 +22,45 @@ Error: There are currently no log messages. Please check the following things:
...
-│     Host     │─────┘
-│       Fluentd│
-└──────────────┘
+┌────────┐                            
+│ Router │                  ┌────────┐
+└────────┘                  │ Logger │
+    │                       └────────┘
+Log file                        │    
+    │                           │    
+    ▼                           ▼
```
Comment (arschles) [inline src/managing-workflow/platform-logging.md]: I think the arrow from logger should point only to `NSQ` and `Redis`

File: src/managing-workflow/platform-logging.md [hunk 4]
```diff
@@ -2,44 +2,16 @@
 
 The logging platform is made up of 2 components - [Fluentd](https://github.com/deis/fluentd) and [Logger](https://github.com/deis/logger).
 
-[Fluentd](https://github.com/deis/fluentd) runs on every worker node of the cluster and is deployed as a [Daemon Set](http://kubernetes.io/v1.1/docs/admin/daemons.html). The Fluentd pods capture all of the stderr and stdout streams of every container running on the host (even those not hosted directly by kubernetes). It then sends this data via the Syslog UDP port (514) to the Logger component.
+[Fluentd](https://github.com/deis/fluentd) runs on every worker node of the cluster and is deployed as a [Daemon Set](http://kubernetes.io/v1.1/docs/admin/daemons.html). The Fluentd pods capture all of the stderr and stdout streams of every container running on the host (even those not hosted directly by kubernetes). Once the log message arrives in our [custom fluentd plugin](https://github.com/deis/fluentd/tree/master/rootfs/opt/fluentd/deis-output) we determine where the message originated.
```
Comment (krancour) [inline src/managing-workflow/platform-logging.md]: Should this line end by saying we filter out messages that aren't from workflow apps?

Comment (krancour) [inline src/managing-workflow/platform-logging.md]: Nevermind... it's below. I should have kept reading.

File: src/managing-workflow/platform-logging.md [hunk 5]
```diff
@@ -50,38 +22,45 @@ Error: There are currently no log messages. Please check the following things:
...
+│ Router │                                        ┌────────┐     ┌─────┐
+└────────┘                                        │ Logger │◀───▶│Redis│
+     │                                            └────────┘     └─────┘
+ Log file                                              ▲                
+     │                                                 │                
+     ▼                                                 │                
+┌────────┐             ┌─────────┐    logs/metrics   ┌─────┐             
+│App Logs│──Log File──▶│ fluentd │───────topics─────▶│ NSQ │             
+└────────┘             └─────────┘                   └─────┘             
+                                                        │
```
Comment (krancour) [inline src/managing-workflow/platform-logging.md]: Shouldn't the touter box point to the fluentd box?

File: src/managing-workflow/platform-monitoring.md [hunk 6]
```diff
@@ -1,38 +1,40 @@
 # Platform Monitoring
 
 ## Description
-
-With the release of Deis Workflow, we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 4 components:
-
+With the release of Workflow Beta4 we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 3 components:
```
Comment (krancour) [inline src/managing-workflow/platform-monitoring.md]: Does talking about the betas even make sense anymore? Could this just be reduced to "Workflow includes..."

File: src/managing-workflow/platform-logging.md (hunk 5 above)
Comment (jchauncey) [inline src/managing-workflow/platform-logging.md]: yes copy and pasting from monodraw sometimes doesnt translate well =p

File: src/managing-workflow/platform-monitoring.md (hunk 6 above)
Comment (jchauncey) [inline src/managing-workflow/platform-monitoring.md]: yeah i removed beta verbage from the logging docs but missed this. It's updated now

**Pull request merged**